diff --git a/config/BuildSystem/nargs.py b/config/BuildSystem/nargs.py
index f2fa577..3f9db45 100644
--- a/config/BuildSystem/nargs.py
+++ b/config/BuildSystem/nargs.py
@@ -303,7 +303,7 @@ class ArgDirList(Arg):
     import os
     self.checkKey()
     if not isinstance(value, list):
-      value = value.split(':')
+      value = value.split(os.pathsep)
     # Should check whether it is a well-formed path
     nvalue = []
     for dir in value:
@@ -361,7 +361,7 @@ class ArgFileList(Arg):
     import os
     self.checkKey()
     if not isinstance(value, list):
-      value = value.split(':')
+      value = value.split(os.pathsep)
     # Should check whether it is a well-formed path
     nvalue = []
     for file in value:
