diff --git a/config/BuildSystem/config/compilerOptions.py b/config/BuildSystem/config/compilerOptions.py
index ab24efa..d397916 100644
--- a/config/BuildSystem/config/compilerOptions.py
+++ b/config/BuildSystem/config/compilerOptions.py
@@ -3,9 +3,9 @@ import os
 import re
 import nargs
 
-re_win32fe_cl    = re.compile(r'win32fe[\s_]+cl')
-re_win32fe_icl   = re.compile(r'win32fe[\s_]+ic[lx]')
-re_win32fe_ifort = re.compile(r'win32fe[\s_]+if(x|ort)')
+re_win32fe_cl    = re.compile(r'win32fe(\.exe)?[\s_]+cl')
+re_win32fe_icl   = re.compile(r'win32fe(\.exe)?[\s_]+ic[lx]')
+re_win32fe_ifort = re.compile(r'win32fe(\.exe)?[\s_]+if(x|ort)')
 
 class CompilerOptions(config.base.Configure):
   def getCFlags(self, compiler, bopt, language):
