diff --git a/config/BuildSystem/config/programs.py b/config/BuildSystem/config/programs.py
index b706ab6..e375a02 100644
--- a/config/BuildSystem/config/programs.py
+++ b/config/BuildSystem/config/programs.py
@@ -92,6 +92,8 @@ AC_OUTPUT
         break
       except RuntimeError:
         self.logPrint('Rejected SEDINPLACE cmd: '+sedcmd)
+      except OSError as e:
+        self.logPrint('Rejected SEDINPLACE cmd (OSError): '+str(e))
     os.unlink(sed1)
     if not status:
         self.logPrint('No suitable SEDINPLACE found')
